WarOfEmpires.CommandHandlers.Tests/Empires/BuildSiegeCommandHandlerTests.cs

```diff
@@ -89,40 +89,6 @@ public sealed class BuildSiegeCommandHandlerTests {
             builder.Player.DidNotReceiveWithAnyArgs().BuildSiege(default, default);
             builder.Context.CallsToSaveChanges.Should().Be(0);
         }
-
-        [TestMethod]
-        public void BuildSiegeCommandHandler_Fails_For_Alphanumeric_Count() {
-            var builder = new FakeBuilder()
-                .BuildPlayer(1);
-
-            var handler = new BuildSiegeCommandHandler(new PlayerRepository(builder.Context));
-            var command = new BuildSiegeCommand("test1@test.com", new List<SiegeWeaponInfo>() {
-                new SiegeWeaponInfo("FireArrows", "A")
-            });
-
-            var result = handler.Execute(command);
-
-            result.Should().HaveError("SiegeWeapons[0].Count", "Invalid number");
-            builder.Player.DidNotReceiveWithAnyArgs().BuildSiege(default, default);
-            builder.Context.CallsToSaveChanges.Should().Be(0);
-        }
-
-        [TestMethod]
-        public void BuildSiegeCommandHandler_Fails_For_Negative_Count() {
```

Comment:
KS Look at after reviewing the model/command/handler, if not clear why testing for negative count would no longer be needed

Reply:
My idea is that all validation that can happen in the model, does happen in the model. And I tried to remove duplicate validation between command handlers and models. So, since the model property here has a regex to check that only 0-6 digits can be input, so the minus sign isn't allowed anymore anyway.